onnxruntime/test/python/transformers/test_flash_attn.py

```diff
@@ -1141,84 +1159,165 @@ def parity_check_gqa_past_no_buff(
     )
 
 
+class TestMHA(unittest.TestCase):
+    def test_packed_mha(self):
+        major, _ = torch.cuda.get_device_capability()
+        if major < 8:
+            return
+        print("-------- TEST PACKED MHA ---------")
+        for b in [5]:
+            for s in [97, 128, 200, 256, 257, 384, 512, 768, 1024, 1025, 2048]:
+                for n in [6]:
+                    for h in [32, 40, 59, 64, 80, 96, 111, 128, 160, 192, 224, 256]:
```

Comment:
Looks like too many combinations 11x12 > 100. How long does it take to run the test? If it is too long, it will impact build pipeline (caused pipeline timeout error)